services/auth/office/serverless.yml

```diff
@@ -0,0 +1,72 @@
+service: officeAuthorizer
```

Comment:
update name to office-authorizer. No other service name follows camelcase in thier name declaration. Also auth-office feels more like a suitable service name?

Reply:
I agree, auth-office feel better. But the current naming of the service (office-authorizer) is more align with the token service, which name is only authorizer. 

I will change the name to office-auth because then it follows the syntax of <service>-api or <service>-ms where the service name comes first.